src/pages/Home/Home.jsx

```diff
@@ -1,11 +1,47 @@
-import Hero from "../../components/Hero";
-import HotProducts from "../../components/HotProducts";
+import Banner from "../../components/Banner";
+import FeaturedProducts from "../../components/FeaturedProducts";
+import Slider from "../../components/Slider";
+
+export default function Home({ changePage }) {
+  let bannerList =
```

Comment:
this could be `const` because you're not redeclaring the variable

Reply:
Resolved with commit https://github.com/FranciscoJAceves/wa-react-2021-q4-internal-capstone-project/pull/2/commits/85b2196fbccda04f0515d13474f1fed084487832